_layouts/quickstart.html

```diff
@@ -9,17 +8,17 @@
 		<div class="row">
 			{% capture versionmaj %}{{ page.version | split: "." | first | times: 100 }}{% endcapture %}
 			{% capture versioni %}{{ page.version | split: "." | last | plus: versionmaj }}{% endcapture %}
-			{% capture latestmaj %}{{ layout.latestversion | split: "." | first | times: 100 }}{% endcapture %}
-			{% capture latesti %}{{ layout.latestversion | split: "." | last | plus: latestmaj }}{% endcapture %}
+			{% capture latestmaj %}{{ site.foreman_version | split: "." | first | times: 100 }}{% endcapture %}
+			{% capture latesti %}{{ site.foreman_version | split: "." | last | plus: latestmaj }}{% endcapture %}
 			{% if page.version == "nightly" or versioni > latesti %}
 			<div class="alert alert-warning" style="margin-top:5px">
 				<p style="margin-bottom: 3px"><strong>Not yet released</strong></p>
-				<p style="margin-bottom: 0">This quickstart guide is for Foreman {{ page.version }}, but <a href="{{ site.baseurl }}manuals/{{ layout.latestversion }}/index.html">{{ layout.latestversion }}</a> is the current stable version.</p>
+				<p style="margin-bottom: 0">This quickstart guide is for Foreman {{ page.version }}, but <a href="{{ site.baseurl }}manuals/{{ site.foreman_version }}/index.html">{{ site.foreman_version }}</a> is the current stable version.</p>
```

Comment:
seems that the link here is incorrect, 
instead of `href="{{ site.baseurl }}manuals/{{ site.foreman_version }}/index.html"`
should be `href="{{ site.baseurl }}manuals/{{ site.foreman_version }}//quickstart_guide.html`

Reply:
or use the fix here: https://github.com/theforeman/theforeman.org/pull/1394 
after I will rebase :P
since it was quickly merged